examples/buttons.html

```diff
@@ -1,8 +1,11 @@
+<!DOCTYPE html>
 <html>
   <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
```

Comment:
Firefox dev complains when there's no content type. 
But otherwise no issue with the removal.